packages/embark-async-wrapper/package.json

```diff
@@ -20,18 +20,18 @@
   "scripts": {
     "build": "cross-env BABEL_ENV=node babel src --extensions \".js,.ts\" --out-dir dist --root-mode upward --source-maps",
     "ci": "npm run qa",
-    "clean": "npx rimraf dist embark-*.tgz package",
+    "clean": "npm run reset",
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint src/",
     "lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
     "package": "npm pack",
     "qa": "npm-run-all lint build package",
-    "reset": "npm run clean && npx rimraf node_modules",
+    "reset": "npx rimraf dist embark-*.tgz package",
     "start": "npm run watch",
-    "typecheck": "tsc",
+    "// typecheck": "tsc",
```

Comment:
What's the purpose of the two forward slashes in this context?

Reply:
It's the closes thing to a "JSON comment" since `package.json` doesn't support [JSON5](https://json5.org/).

The idea is that eventually modules with these "comments" will be converted to typescript and then we can "uncomment" those lines.  If the transition to typescript is long delayed, we can just removed those lines at some point.